### PR TITLE
Update react-native-libraries.json with react-native-otp-entry lib

### PR DESCRIPTION
**Pull request by anday013**
# 📝 Why & how
Update react-native-libraries.json with react-native-otp-entry lib

# ✅ Checklist

- [x] Added library to **`react-native-libraries.json`**
- [ ] Updated library in **`react-native-libraries.json`**

- [ ] Documented in this PR how to use the feature or replicate the bug.
- [ ] Documented in this PR how you fixed or created the feature.

### CODE REVIEW

**Review by Simek — COMMENTED** [review]
Hello @anday013, thank you for adding new library to the directory! 👍 

Let's move the library to the end of file, since we use the order for "Recently added" filter on the website. Additionally, `npmPkg` field can be removed since repository matches package name, this field is mostly for scoped packages.

**Comment by anday013** [comment]
@Simek thank you for the feedback, I've addressed the issues :)

File: react-native-libraries.json [hunk 1]
```diff
@@ -10016,5 +10016,12 @@
     ],
     "ios": true,
     "android": true
+  },
+  {
+    "githubUrl": "https://github.com/anday013/react-native-otp-entry",
+    "images": ["https://github.com/anday013/react-native-otp-entry/blob/main/otp_demo.gif"],
+    "ios": true,
+    "android": true,
+    "expo": true,
```
Comment (Simek) [inline react-native-libraries.json:10025]:
```suggestion
    "expo": true
```

Thanks! Currently CI fails due to tailing comma.

File: react-native-libraries.json [hunk 2]
```diff
@@ -10018,6 +10018,12 @@
     "android": true
   },
   {
+    "githubUrl": "https://github.com/anday013/react-native-otp-entry",
+    "images": ["https://github.com/anday013/react-native-otp-entry/blob/main/otp_demo.gif"],
+    "ios": true,
+    "android": true,
+    "expo": true
+  },
```
Comment (Simek) [inline react-native-libraries.json:10026]:
```suggestion
  },
  {
```

Merge resolve fix.

**Review by Simek — APPROVED** [review]
Thank you for the updates, let's ship it! 🚀

**Pull request merged**